eagleEvents/templates/event.html.j2

```diff
@@ -20,32 +32,89 @@
                         </tr>
                     </thead>
                     <tbody>
-                        <tr>
-                            <td>Event 1</td>
-                            <td>Eva</td>
-                            <td>04/12/2018</td>
-                        </tr>
-                        <tr>
-                            <td>Event 2</td>
-                            <td>Jacie</td>
-                            <td>08/22/2018</td>
-                        </tr>
-                        <tr>
-                            <td>Event 3</td>
-                            <td>David</td>
-                            <td>01/23/2019</td>
+                        {% for event in events %}
+                        <tr class = "click-row" data-href="modifyEvent/ + {{event.id}}">
```

Comment:
There is no need for a "+" sign here. I merged this into my branch to test it and it works if you do this:

<tr class = "click-row" data-href="modifyEvent/{{event.id}}">